setup.py

```diff
@@ -46,7 +46,7 @@
 
 extras_require = {
     'docs': [
-        'Sphinx>=1.4.2',
+        'Sphinx>=1.6.2',
```

Comment:
Yes I got another issue with the previous version of sphinx. See https://github.com/sphinx-doc/sphinx/issues/3212